integration-test/src/main/java/com/sequenceiq/it/cloudbreak/client/EnvironmentTestClient.java

```diff
@@ -36,6 +38,10 @@
         return new EnvironmentGetAction();
     }
 
+    public Action<EnvironmentTestDto, EnvironmentClient> describeByCrn() {
```

Comment:
could you please change for this: `EnvironmentInternalGetAction` or remove one of them?

Reply:
why internal? both of them are worth to test and both of them can be used by anyone

Reply:
ohh, i see now, but my method is not using internalcrn, and that's the real use case in my test

Reply:
Yes, you are right. Ok, it will be ok